core/amber/src/main/python/core/architecture/sendsemantics/partitioner.py

```diff
@@ -20,6 +20,9 @@ def add_tuple_to_batch(
     ) -> Iterator[typing.Tuple[ActorVirtualIdentity, typing.List[Tuple]]]:
         pass
 
+    def add_state_to_batch(self, state: State):
```

Comment:
I have a feeling this method can be implemented in the base (so that all child partitioners do not need to implement it again).

Reply:
Moved to base.